Formula/fermyon-spin.rb

```diff
@@ -0,0 +1,49 @@
+class FermyonSpin < Formula
+  desc "Open-source tool for building and running serverless WebAssembly applications"
+  homepage "https://spin.fermyon.dev/"
+  url "https://github.com/fermyon/spin.git",
+    tag:      "v1.3.0",
+    revision: "9fb8256d1380a046414b22bf2c17d1543f5029e4"
+  license "Apache-2.0" => { with: "LLVM-exception" }
+
+  depends_on "rust" => [:build, :test]
+  depends_on "rustup-init" => [:build, :test]
+
+  uses_from_macos "zlib"
+
+  on_linux do
+    depends_on "pkg-config" => :build
+  end
+
+  def install
+    system "#{Formula["rustup-init"].bin}/rustup-init", "-qy", "--no-modify-path"
```

Comment:
This should be build with `rust` instead

Reply:
Why wouldn't the normal rust do this?

Reply:
@SMillerDev `rust` cannot add targets and it doesn't seem that wasm targets are packaged with the homebrew `rust` formula. Using `rustup` is the next best bet and does appear to be a convention, given [`rustfmt`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/rustfmt.rb) and [`cargo-zigbuild`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/cargo-zigbuild.rb#L36)